include/circt/Dialect/Pipeline/Pipeline.td

```diff
@@ -126,82 +127,181 @@ def PipelineOp : Op<Pipeline_Dialect, "pipeline", [
       return &region->front();
     }
 
+    // Returns the internal inputs of this pipeline.
+    ValueRange getInnerInputs() { return getBodyBlock()->getArguments().drop_back(1); }
+
+    // Returns the internal enable signal of this pipeline.
+    Value getInnerEnable() {
+      return getBodyBlock()->getArguments().back();
+    }
   }];
 }
 
 
-def PipelineStageOp : Op<Pipeline_Dialect, "stage", [
-    HasParent<"PipelineOp">
+def StageSeparatingOp : Op<Pipeline_Dialect, "ss", [
+    HasParent<"PipelineOp">,
+    StageLike
   ]> {
-  let summary = "Pipeline pipeline stage.";
+  let summary = "Pipeline **s**tage **s**eparating operation.";
```

Comment:
Are the asterisks here a typo, or do they have meaning?

Reply:
This will eventually materialize into autogenerated operation documentation (like [this](https://circt.llvm.org/docs/Dialects/Pipeline/#pipelinestageregister-circtpipelinepipelinestageregisterop)), which supports markdown formatting.
Hence, this will be printed as:

Pipeline **s**tage **s**eparating operation